en/docs/learn/analytics/analyzing-apim-statistics-with-batch-analytics/viewing-api-statistics.md

```diff
@@ -498,3 +487,25 @@ The status of the APIs (all API versions) represented in a tabular view.
 </html>
 
 The availability of APIs statistics is directly related to the [Availability of APIs (health monitoring)](../../../../learn/analytics/managing-alerts-with-real-time-analytics/alert-types/#availability-of-apis-api-health-monitoring) alert type.
+
+## Date Range Picker
+
+This can be used to configure the **time period** and **granularity mode** of the data should be visualized or reported in the widgets listed above. Here the aggregated data to the selected granularity will be visualized or reported in widgets.
+
+The following diagram shows the configurable options in this widget.  
+
+<html>
+<body>
+<img src="../../../../assets/img/learn/publisher-date-time-picker.png" width="500"/>
+</body>
+</html>
+
+The time period can be defined, either choosing from `Quick Ranges` or defining `Custom Ranges`. For the quick ranges `1 Min`, `15 Min`, `1 Hour`, `1 Day`, `7 Days`, `1 Month`, `3 Months`, `6 Months`, and `1 Year` options are provided.
+For `Custom Ranges` users can define the start date as `From` and end date as `To`.
+
+For `Quick Ranges` the **granularity mode** selected as default and for `Custom Ranges`, users can choose from `second`, `minute`, `hour`, `day`, `month` and `year`.
+
+Generally the widgets will visualize or report the data retrieved during the initialization. But if a user required to update the widget data with time, can toggle the `sync button` at the upper right corner of the diagram.
```

Comment:
```suggestion
Generally, the widgets will visualize or report the data retrieved during the initialization. But if a user is required to update the widget with new data since the widget's initial load, they can toggle the `sync button` at the upper right corner of the diagram.
```

Reply:
We also need to insert an image highlighting this button.

Reply:
Resolved with bea00a7